script.js

```diff
@@ -0,0 +1,248 @@
+            console.log(count)
+            this.badCoinB.update();
+
+        }
+    }
+}
+
+class Player {
+    constructor(game) {
+        this.game = game;
+        this.screen = game.screen;
+        this.keyboarder = new Keyboarder();
+        this.location = {
+            x: 230,
+            y: 230,
+            sizeX: 80,
+            sizeY: 20
+        }
+    }
+    draw() {
+        this.screen.fillStyle = "#FF8800";
+        var sizeX = this.location.sizeX;
+        var sizeY = this.location.sizeY;
+        var leftX = this.location.x
+        var leftY = this.location.y
+        this.screen.fillRect(leftX, leftY, sizeX, sizeY);
+    }
+    update() {
+
+        if (this.keyboarder.isDown(Keyboarder.KEYS.LEFT) && this.keyboarder.isDown(Keyboarder.KEYS.SPACE) && this.location.x > 155) {
+            this.location.x -= 7;
+
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.RIGHT) && this.keyboarder.isDown(Keyboarder.KEYS.SPACE) && this.location.x < 264) {
+            this.location.x += 7;
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.DOWN) && this.keyboarder.isDown(Keyboarder.KEYS.SPACE) && this.location.y < 325) {
+            this.location.y += 7;
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.UP) && this.keyboarder.isDown(Keyboarder.KEYS.SPACE) && this.location.y > 155) {
+            this.location.y -= 7;
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.LEFT) && this.location.x > 155) {
+            this.location.x -= 3;
+
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.RIGHT) && this.location.x < 264) {
+            this.location.x += 3;
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.DOWN) && this.location.y < 325) {
+            this.location.y += 3;
+        } else if (this.keyboarder.isDown(Keyboarder.KEYS.UP) && this.location.y > 155) {
+            this.location.y -= 3;
+        }
+
+
+    }
+}
+
+class Coin {
+    constructor(game) {
+        this.game = game;
+        this.screen = game.screen;
+        this.location = {
+            x: (Math.floor(Math.random() * 155) + 155),
+            y: ((Math.floor(Math.random() * 155)) + 155)
+        }
+    }
+    draw() {
+        this.screen.fillStyle = "#FFDD00";
+        var sizeX = 20;
+        var sizeY = 20;
+        this.screen.fillRect(this.location.x, this.location.y, sizeX, sizeY);
+    }
+    update() {
+        this.location.x = (Math.floor(Math.random() * 155)) + 155;
+        this.location.y = (Math.floor(Math.random() * 155)) + 155;
+    }
+}
+
+
+class BadCoin {
+    constructor(game, direction) {
+        this.game = game;
+        this.screen = game.screen;
+        this.location = {
+            x: (Math.floor(Math.random() * 155) + 155),
+            y: (0)
+        }
+
+    }
+    draw() {
+        this.screen.fillStyle = "#FF0000";
+        var sizeX = 20;
+        var sizeY = 20;
+        this.screen.fillRect(this.location.x, this.location.y, sizeX, sizeY);
+
+    }
+    update() {
```

Comment:
Even though `update` is the wrong name here, I still really like how you reused your `BadCoin` as it left the screen.

![bitmoji](https://render.bitstrips.com/v2/cpanel/359644fb-c5df-4412-9bc7-ae838696ef14-bb69060c-eccd-4c08-a1d6-595d0b37616d-v1.png?transparent=1&palette=1&width=246)